tutorials/nservicebus-101/lesson-4/tutorial.md

```diff
@@ -0,0 +1,212 @@
+---
+title: "NServiceBus 101 Lesson 4: Publishing events"
+---
+
+So far in this course, we have only sent **commands** – one-way messages from a sender to a specific receiver. There's another type of message we have yet to cover called an **event**. In many ways events are just like commands. They're simple classes and you deal with them in much the same way. But from an architectural standpoint commands and events are *polar opposites*. This creates a useful dichotomy. We can take advantage of the properties of events to open up new possibilities in how we design software.
+
+In this lesson, we'll learn all about publishing events, and more importantly, what that allows us to do.
+
+
+## Objectives
+
+By the end of this lesson, you will have learned:
+
+ * The difference between commands and events
+ * How the Publish/Subscribe pattern helps to create more maintainable code
+ * How to define and name events
+ * How to publish an event
+ * How to subscribe to an event
+
+
+## What is an event?
+
+An **event** is another type of message that is published to multiple receivers, unlike a command which is sent to one receiver. Let's take a look at the formal definitions for commands and events:
+
+{{INFO:
```

Comment:
i think re-defining event and command makes it problematic with this https://docs.particular.net/nservicebus/concept-overview#message

use an include to use the same definition